From dceab4127e60691abbe708afb58778f411c2707d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 28 Mar 2013 15:54:35 +0000
Subject: [PATCH 0328/1000] LogService.exe V1.0.4.6 massaiSNMP.dll V1.0.4.6
 Mantis  0026620: LogService: HPOpenView-setup for dnata Kiosk new
 event-/comp-configuration for dnata hp openview

git-svn-id: svn://localhost/SelfServiceCommon/trunk@493 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
index fb69380f..169a5895 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
+++ b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
@@ -3350,4 +3350,5 @@
 999999;massai.setup.Applications.LX;[SNMP:#:1:30];Applications::LX
 999999;massai.setup.Applications.CX;[SNMP:#:1:30];Applications::CX
 999999;massai.setup.Applications.VS;[SNMP:#:1:30];Applications::VS
+999999;massai.setup.Applications.IX;[SNMP:#:1:30];Applications::IX
 
-- 
2.41.0.windows.1

